### ISSUE TITLE
SDK 3.1.300 - Microsoft.AspNetCore.App Version cannot update from 3.1.2 to 3.1.3

### ISSUE THREAD

**Issue by GaryTian-1984** [dotnet/core#4726]
SDK 3.1.300 - Microsoft.AspNetCore.App Version cannot update from 3.1.2 to 3.1.3

Windows 10 Enterprise 1909
SDK version 3.1.300
VS 2019 Enterprise 16.6.0

I was installed the latest version of SDK 3.1.4 and run dotnet --version is show 3.1.300.
But I created new asp.net core MVC project, the version of Microsoft.AspNetCore.App is still 3.1.2 even the 3.1.3 installed in my computer. Means the new version of Microsoft.AspNetCore.App cannot be used.

In my computer the location of Microsoft.AspNetCore.App is stored in C:\Program Files\dotnet\packs\Microsoft.AspNetCore.App.Ref. 3 versions folder in this location. '3.0.1','3.1.2' and '3.1.3'.

**Comment by ttutko** [comment]
I think I am seeing something similar to this. (If it's not the same, please move to a separate issue.) 

I upgraded Visual Studio 2019 Enterprise from 16.5.latest to the newly released 16.6.0. After a successful install, I am now unable to build aspnetcore or dotnetcore console apps. If I do a `dotnet build` I get the following error:

"error : NU1101: Unable to find package Microsoft.AspNetCore.App.Ref. No packages exist with this id in sources(s): Microsoft Visual Studio Offline Packages, <private repo>"

I tried our existing solution that I am working on consisting of approximately 30 projects as well as just a basic new dot net core console application from within Visual Studio. I then went to the command line to try building the brand new empty console application and got the same error so that's why I am filing an issue here.

**Note:** I work in an offline air-gapped environment and I updated Visual Studio following the same offline steps I have always used. 

I'm going to try manually installing the SDK 3.1.300. I don't know if this is an issue with Visual Studio's packaging of dotnet core sdk 3.1.300 or an issue with the sdk itself. `dotnet --list-sdks` shows that 3.1.300 is installed and `dotnet --list-runtimes` shows that 3.1.4 is installed.

**Comment by ttutko** [comment]
Reporting back that manually downloading and installing the SDK resolved the issue _when doing a build from the command line_, however VS2019 16.6 is still failing to find the Microsoft.AspNetCore.App.ref package. What is strange is that despite seeing "3.1.300" listed when doing `dotnet --list-sdks` prior to manually installing the sdk separately, the install acted like it wasn't installed in that it didn't ask me to uninstall/repair. Since I am not sure where the issue lies I'm going to also open up an issue with the VS team.

**Comment by ttutko** [comment]
Exiting visual studio and restarting it seems to have fixed the visual studio build issue.

**Comment by GaryTian-1984** [comment]
@ttutko Yes, I found this issue is should be a bug of VS 2019 16.6.0. In my another computer, befor VS 2019 upgrade(VS 2019 16.5) the version of  Microsoft.AspNetCore.App is 3.1.3 in my projects, but when upgrade to 16.6.0 the version are changed from 3.1.3 to 3.1.2 in same projects.

**Comment by ttutko** [comment]
It seems that the Visual Studio team has identified the issue and has a fix in the works as per my bug report on their feedback system: [Issue with .Net Core SDK 3.1.300 installation after upgrading Visual Studio](https://developercommunity.visualstudio.com/content/problem/1050924/issue-with-net-core-sdk-31300-installation-after-u.html).
